security.rst

```diff
@@ -1838,41 +1837,68 @@ To enable logging out, activate the  ``logout`` config parameter under your fire
             $mainFirewall = $security->firewall('main');
             // ...
             $mainFirewall->logout()
-                // the argument can be either a route name or a path
-                ->path('app_logout')
+                ->path('/logout')
 
                 // where to redirect after logout
                 // ->target('app_any_route')
             ;
         };
 
-Next, you need to create a route for this URL (but not a controller):
+Symfony will then un-authenticate users navigating to the configured ``path``,
+and redirect them to the configured ``target``. You can generate URLs to this
+path using the ``_security_<firewallname>`` route name (e.g. ``_security_main``).
```

Comment:
```suggestion
path using the ``_logout_<firewallname>`` route name (e.g. ``_logout_main``).
```
I came across this change when updating the recipes for my app, and noticed that the documentation is not yet merged, and it is not fully correct. Using `_logout_main` works for me.

Reply:
Good catch thanks!

I'm not even sure this needs to be documented though as we should use the LogoutUrlGenerator (see https://github.com/symfony/symfony-docs/pull/19000#discussion_r1360445595).

I'll push your edit in the meantime.

Reply:
Ok, thanks for updating. How do you expect to use the `LogoutUrlGenerator` in a Twig template? That's where I had to change the route name...

Reply:
You should be able to use `logout_path` or `logout_url`; see https://symfony.com/doc/current/reference/twig_reference.html#logout-path

Reply:
That's awesome, I didn't know about this Twig function! Thanks for showing me 😄